applications/openshift/api-server/var_apiserver_encryption_type.var

```diff
@@ -11,4 +11,4 @@ operator: equals
 interactive: false
 
 options:
-    default: 'aescbc'
+    default: 'aescbc|aesgcm'
```

Comment:
Need to understand if this will pass on 4.12 and earlier deployments. The CIS guidance only recommends using this type on 4.13 and later.

We may need to split this rule into two rules for each version (pre-4.13 and post-4.13).

Reply:
I think this will work.
The variable is used with a `pattern match` operation:
```xml
<ind:yamlfilecontent_state id="oval:ssg-state_api_server_encryption_provider_cipher:ste:1" version="1">
  <ind:value datatype="record" entity_check="all">
    <oval-def:field name="#" operation="pattern match" var_ref="oval:ssg-var_apiserver_encryption_type:var:1"/>
  </ind:value>
</ind:yamlfilecontent_state>
```